pkg/elemental/elemental.go

```diff
@@ -17,20 +17,26 @@ limitations under the License.
 package elemental
 
 import (
+	"context"
 	"errors"
 	"fmt"
-	"github.com/kairos-io/kairos-sdk/types"
 	"os"
 	"path/filepath"
 	"syscall"
 
+	"github.com/kairos-io/kairos-sdk/types"
+
+	"github.com/containerd/containerd/archive"
```

Comment:
nit: wondering if we could sue any other existing dep instead of bringing containerd as its a pita to deal with it when there are upgrades.